apps/explorer/components/Layout/Navbar.tsx

```diff
@@ -16,25 +16,32 @@ type Props = {
 
 export function Navbar({ appName, network, homeHref, children }: Props) {
   return (
-    <div className={cx("py-3 z-10 bg-white dark:bg-graydark-50 border-b border-gray-200 dark:border-graydark-200",
-      network === 'mainnet' ? (
-        "border-t-4  border-t-green-500 dark:border-t-green-400"
-      ) : '',
-      network === 'zen' ? (
-        "border-t-4  border-t-amber-500 dark:border-t-amber-400"
-      ) : '',
-    )}>
+    <div
+      className={cx(
+        'py-3 z-10 bg-white dark:bg-graydark-50 border-b border-gray-200 dark:border-graydark-200',
+        network === 'mainnet'
+          ? 'border-t-4  border-t-green-500 dark:border-t-green-400'
+          : '',
```

Comment:
woah how did those " get in there haha.

Reply:
Oh, good point. I have no idea. Could be code that hasn't changed since you've changed to single quotes? Likely picked it up with an auto-save.